### PR TITLE
Code has been refactore to be more readable for all users

### PR DESCRIPTION
**Pull request by RobertHilla**
Elements which has been deleted :
1. Managers
2. Elements that weren't used

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: web/src/main/java/com/isa/service/ApiInitialization.java [hunk 1]
```diff
@@ -21,26 +21,26 @@
...
+    private PlaceService placeService;
 
 
     @PostConstruct
     public void apiInitialization() throws IOException {
 
-        organizersManager.setRelations(converter.buildClientRequest(constantValuesBean.getOrganizersApi()));
-        placeManager.setRelations(converter.buildClientRequest(constantValuesBean.getPlaceApi()));
-        eventManager.setRelationsToEntity(converter.buildClientRequest(constantValuesBean.getEventsApi()));
+        organizerService.setRelations(generateRequest.buildClientRequest(constantValuesBean.getOrganizersApi()));
```
Comment (rafalmisiak) [inline web/src/main/java/com/isa/service/ApiInitialization.java]: setRelations - nazwa a znaczenie

**Comment by isacademyci** [comment]
Build triggered for original commit.

**Comment by isacademyci** [comment]
Build started for original commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for original commit.

**Comment by isacademyci** [comment]
Build started for original commit.

**Comment by isacademyci** [comment]
Build finished.

File: web/src/main/java/com/isa/domain/api/EventApi.java [hunk 2]
```diff
@@ -25,6 +25,11 @@
     private WeblinkExternal weblinkExternal;
 
 
+    @JsonProperty("schedule")
+    @JsonIgnore
```
Comment (rafalmisiak) [inline web/src/main/java/com/isa/domain/api/EventApi.java]: dlaczego ignore i property jednoczesnie ?

**Comment by isacademyci** [comment]
Build triggered for original commit.

**Comment by isacademyci** [comment]
Build started for original commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**